test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -4065,6 +4065,31 @@ def forward(self, src, index):
         index = torch.tensor([[0, 1], [0, 1], [0, 1]], dtype=torch.int64)
         self.run_test(ScatterModel(), (src, index))
 
+    @skipIfUnsupportedMinOpsetVersion(16)
+    def test_scatter_add_index_not_unique(self):
+        class ScatterModel(torch.nn.Module):
+            def forward(self, input, indices, values):
+                return input.scatter_add(1, indices, values)
+
+        input = torch.tensor([[0.0, 0.0, 0.0], [0.0, 0.0, 0.0], [0.0, 0.0, 0.0]])
+        indices = torch.tensor([[0, 0], [1, 1], [2, 2]], dtype=torch.int64)
+        values = torch.tensor([[1.0, 1.1], [2.0, 2.1], [3.0, 3.1]])
+        self.run_test(ScatterModel(), input_args=(input, indices, values))
+
+        @torch.jit.script
+        def scatter_sum(src: Tensor, index: Tensor):
+            size = src.size()
+            out = torch.zeros(size, dtype=src.dtype)
+            return out.scatter_add_(1, index, src)
+
+        class ScatterModel(torch.nn.Module):
```

Comment:
I just realized this, but are we redefining the same name here? If so could you help us break out of this pattern by splitting the test case? Or @BowenBao are you tackling this in your parameterization pr?

Reply:
There are too many similar patterns like this in the file (this one is mostly taken from `test_scatter_add`). if someone is working on this with a PR, I think we just leave it as is here and have it handled by the other PR which fixes this kind of inconsistencies altogether.

Reply:
SGTM